simapp/app.go

```diff
@@ -373,7 +373,7 @@ func NewSimApp(
 
 	// Fetch JWK
 	ctx := app.BaseApp.NewUncachedContext(true, tmproto.Header{})
-	app.ZKAuthKeeper.FetchJWK(ctx)
+	go app.ZKAuthKeeper.FetchJWK(ctx)
```

Comment:
It seems that goroutine is also no longer needed.

Reply:
```suggestion
	app.ZKAuthKeeper.FetchJWK(ctx)
```

Reply:
Oops, thank you!